service/release.go

```diff
@@ -194,3 +203,30 @@ func (s *ReleaseService) SendReleaseNotification(ctx context.Context, projectID,
 
 	return nil
 }
+
+func (s *ReleaseService) deleteGithubRelease(ctx context.Context, projectID, releaseID, authUserID uuid.UUID) error {
+	rls, err := s.repo.Read(ctx, projectID, releaseID)
```

Comment:
I will also change order as you suggested here https://github.com/jan-zabloudil/release-manager/pull/130